task/02-numbers-tasks.js

```diff
@@ -202,7 +209,10 @@ function roundToPowerOfTen(num, pow) {
  *   17 => true
  */
 function isPrime(n) {
-    throw new Error('Not implemented');
+    for(let i=2; i<n; i++){
+        if(n%i === 0) return false;
```

Comment:
JFYI: use braces in each `for`/`while`/`do ... while`/`if` operator. You will make less mistakes.